release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -203,17 +203,29 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
-[id="ocp-4-6-operators"]
-=== Operators
+[id="ocp-4-6-storage"]
+=== Storage
 
-[id="ocp-4-6-metering-operator"]
-==== Configuring a retention period of metering Reports
+[id="ocp-4-6-csi-driver-cso"]
+==== CSI drivers now managed by the Cluster Storage Operator
+The Container Storage Interface (CSI) Driver Operators and drivers for xref:../storage/container_storage_interface/persistent-storage-csi-ebs.adoc[AWS Elastic Block Store (EBS)], xref:../storage/container_storage_interface/persistent-storage-csi-ovirt.adoc[Red Hat Virtualization (oVirt)], and xref:../storage/container_storage_interface/persistent-storage-csi-ovirt.adoc[OpenStack Manila shared file system service]  are now managed by the Cluster Storage Operator in the {product-title} back end.
```

Comment:
possible extra space before "are now managed"

Each of the links need a target anchor ID after the .adoc

We should avoid using "back end" per the IBM Style Guide. Perhaps we can just say, "managed by the Cluster Storage Operator in {product-title}."

Reply:
Thanks, @ahardin-rh - these were great catches! I've updated per your suggestions. 🙇